src/libdrakvuf/libdrakvuf.h

```diff
@@ -265,6 +265,7 @@ struct drakvuf_trap
     void(*ah_cb)(drakvuf_t, drakvuf_trap_t*);
 };
 
+typedef uint16_t domid_t;
```

Comment:
I don't think we need this typedef.

Reply:
Compilation fails, if will be not defined with the following error:
```
libdrakvuf.h:412:1: error: unknown type name ‘domid_t’; did you mean ‘id_t’?
  412 | domid_t drakvuf_get_dom_id(drakvuf_t drakvuf) NOEXCEPT;
```
domid_t is defined in <xen.h>, this header file is only included by xen_helper/xen_helper and xen_helper in turn is only included in libdrakvuf/drakvuf.c and libdrakvuf/private.h 

Do you see another solution? (Including xen_helper directly didn't seem apt to me)

Reply:
Return uint16_t instead of domid_t

Reply:
Done. Newly added function `drakvuf_get_dom_id(...)` returns uint16_t now. Thanks for reviewing!